### ISSUE TITLE
TypeError for Master

### ISSUE THREAD

**Issue by sw2030** [ranjanan/DIA.jl#21]
```
julia> using DIA
[ Info: Precompiling DIA [aa37cf10-42a5-11e9-3f27-95cdb26253a6]
ERROR: LoadError: LoadError: TypeError: in Type{...} expression, expected UnionAll, got typeof(DIA.SparseMatrixCSC)
Stacktrace:
 [1] top-level scope at none:0
 [2] include at ./boot.jl:326 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1038
 [4] include at ./sysimg.jl:29 [inlined]
 [5] include(::String) at /home/sw2030/.julia/packages/DIA/MJyhL/src/DIA.jl:1
 [6] top-level scope at none:0
 [7] include at ./boot.jl:326 [inlined]
 [8] include_relative(::Module, ::String) at ./loading.jl:1038
 [9] include(::Module, ::String) at ./sysimg.jl:29
 [10] top-level scope at none:2
 [11] eval at ./boot.jl:328 [inlined]
 [12] eval(::Expr) at ./client.jl:404
 [13] top-level scope at ./none:3
in expression starting at /home/sw2030/.julia/packages/DIA/MJyhL/src/convert.jl:6
in expression starting at /home/sw2030/.julia/packages/DIA/MJyhL/src/DIA.jl:10
ERROR: Failed to precompile DIA [aa37cf10-42a5-11e9-3f27-95cdb26253a6] to /home/sw2030/.julia/compiled/v1.1/DIA/5pxLb.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1197
 [3] _require(::Base.PkgId) at ./loading.jl:960
 [4] require(::Base.PkgId) at ./loading.jl:858
 [5] require(::Module, ::Symbol) at ./loading.jl:853
```

**Comment by ranjanan** [comment]
Right, will fix right away

**Comment by ranjanan** [comment]
Fixed now via 9e5e9c9
